packages/_debian-common/changelog

```diff
@@ -1,9 +1,12 @@
-libsuperiormysqlpp (0.3.3) UNRELEASED; urgency=medium
+libsuperiormysqlpp (0.3.4) UNRELEASED; urgency=medium
```

Comment:
You don't need to increment patch version because this version (0.3.3) is not released yet.

Reply:
ACK